### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by OoogleBoogle**
Set up redux. @kaeside @cjjujubee please review.

### CODE REVIEW

**Comment by conniejew** [comment]
looks good to me

**Comment by OoogleBoogle** [comment]
Added the saving user actions. Needs endpoint to save a user to DB. Ready for review @cjjujubee @kaeside

File: redux/actions/user.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return fetch('ENDPOINT', {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        firstName: user.firstName,
+        lastName: user.lastName,
+        email: user.lastName,
```
Comment (conniejew) [inline redux/actions/user.js]: typo !!!!! :)

Comment (OoogleBoogle) [inline redux/actions/user.js]: BALLS!

**Pull request merged**